From dfd0775ce1bf6c1702f17c36a893110401b81068 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 4 Nov 2011 10:22:59 +0100
Subject: [PATCH] 45ifcfg/write-ifcfg.sh: check for existance of
 /tmp/net.$netif.override

check for existance of /tmp/net.$netif.override before sourcing it

https://bugzilla.redhat.com/show_bug.cgi?id=696980
---
 modules.d/45ifcfg/write-ifcfg.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/45ifcfg/write-ifcfg.sh b/modules.d/45ifcfg/write-ifcfg.sh
index f5989ef..b7148d8 100644
--- a/modules.d/45ifcfg/write-ifcfg.sh
+++ b/modules.d/45ifcfg/write-ifcfg.sh
@@ -27,7 +27,7 @@ for netif in $IFACES ; do
 	else
 	    echo "BOOTPROTO=none" 
         # If we've booted with static ip= lines, the override file is there
-	    . /tmp/net.$netif.override 
+	    [ -e /tmp/net.$netif.override ] && . /tmp/net.$netif.override 
 	    echo "IPADDR=$ip"
 	    echo "NETMASK=$mask"
 	    [ -n "$gw" ] && echo "GATEWAY=$gw"
